### ISSUE TITLE
pH and ORP do not show values in either homebridge accessory or Home (iOS).  Could it be possible that they need to be defined as something else other than filter maintenance?

### ISSUE THREAD

**Issue by oserani** [GurYN/homebridge-iopool#2]
**Describe The Bug:**

**To Reproduce:**

**Expected behavior:**

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by GurYN** [comment]
Hi,

The values can be shown in the data page of the accessories.

Example below for pH:

<img width="517" alt="image" src="https://github.com/GurYN/homebridge-iopool/assets/577308/48c448aa-f626-47bc-be2a-688aa1117a3e">

<img width="784" alt="image" src="https://github.com/GurYN/homebridge-iopool/assets/577308/11e44b2f-533b-4ff3-adac-52dbd72966e7">

**Comment by oserani** [comment]
yes - thanks, I had noticed that, but it’s not visible in HomeKit (iOS).

**Comment by RRoyceus** [comment]
I have the same problem.  Home kit shows the temperature, but not the PH or ORP.  Home kit shows “! No Remote Acess”[
